src/schema/v2/artwork/__tests__/artwork.test.js

```diff
@@ -2574,50 +2574,186 @@ describe("Artwork type", () => {
+              expect(data).toEqual({
+                artwork: {
+                  onlyShipsDomestically: false,
+                },
+              })
+            })
+          })
         })
       })
     })
 
-    it("is false when work ships worldwide", () => {
-      artwork.domestic_shipping_fee_cents = 1000
-      artwork.international_shipping_fee_cents = 1000
-      return runQuery(query, context).then((data) => {
-        expect(data).toEqual({
-          artwork: {
-            onlyShipsDomestically: false,
-          },
+    describe("when artsy domestic shipping disabled", () => {
+      beforeEach(() => {
+        artwork.process_with_artsy_shipping_domestic = false
+      })
+
+      describe("when domestic_shipping_fee_cents and international_shipping_fee is null", () => {
+        beforeEach(() => {
+          artwork.domestic_shipping_fee_cents = null
+          artwork.international_shipping_fee_cents = null
+        })
+
+        it("returns false", () => {
+          return runQuery(query, context).then((data) => {
+            expect(data).toEqual({
+              artwork: {
+                onlyShipsDomestically: false,
+              },
+            })
+          })
+        })
+      })
+
+      describe("when domestic_shipping_fee_cents is present", () => {
+        beforeEach(() => {
+          artwork.domestic_shipping_fee_cents = 100
+          artwork.international_shipping_fee_cents = null
+        })
+
+        it("returns true", () => {
+          return runQuery(query, context).then((data) => {
+            expect(data).toEqual({
+              artwork: {
+                onlyShipsDomestically: true,
+              },
+            })
+          })
+        })
+      })
+
+      describe("when only international_shipping_fee_cents is present", () => {
+        beforeEach(() => {
+          artwork.domestic_shipping_fee_cents = null
+          artwork.international_shipping_fee_cents = 100
+        })
+
+        it("returns false", () => {
+          return runQuery(query, context).then((data) => {
+            expect(data).toEqual({
+              artwork: {
+                onlyShipsDomestically: false,
+              },
+            })
+          })
+        })
+
+        describe("when only artsy international shipping is present", () => {
+          beforeEach(() => {
+            artwork.domestic_shipping_fee_cents = null
+            artwork.international_shipping_fee_cents = null
+            artwork.artsy_shipping_international = true
+          })
+
+          it("returns false", () => {
+            return runQuery(query, context).then((data) => {
+              expect(data).toEqual({
+                artwork: {
+                  onlyShipsDomestically: false,
+                },
+              })
+            })
+          })
+        })
+
+        describe("when free shipping worldwide", () => {
+          beforeEach(() => {
+            artwork.domestic_shipping_fee_cents = 0
```

Comment:
So yea, I want to see a spec with `artwork.domestic_shipping_fee_cents = 0 and artwork.international_shipping_fee_cents = nil` return `true` here.